Sources/Utility/PkgConfig.swift

```diff
@@ -110,11 +138,15 @@ public struct PkgConfig {
     static func locatePCFile(
         name: String,
         customSearchPaths: [AbsolutePath],
-        fileSystem: FileSystem
+        fileSystem: FileSystem,
+        diagnostics: DiagnosticsEngine
     ) throws -> AbsolutePath {
         // FIXME: We should consider building a registry for all items in the
         // search paths, which is likely to be substantially more efficient if
         // we end up searching for a reasonably sized number of packages.
+        if pkgConfigSearchPaths == [] {
```

Comment:
I think we should explicitly check for execution errors. I don't know if its valid for pkg-config to return zero paths.

Reply:
@aciidb0mb3r Sounds good to me. In this case, should we move from a computed property to a function? That way we can throw an error if the execution fails.

Reply:
Yeah and probably cache the execution result in a variable.

Reply:
Awesome, got this re-worked into a `struct`.